controllers/util/collection_util.go

```diff
@@ -59,6 +60,31 @@ func CreateCollection(cloud string, collection string, numShards int64, replicat
 	return success, err
 }
 
+// CreateCollecttionAlias to request the creation of an alias to one or more collections
+func CreateCollecttionAlias(cloud string, alias string, aliasType string, collections []string, namespace string) (err error) {
```

Comment:
type in "Collection" in both method and comment above.

Reply:
bah! good catch